pelita/utils.py

```diff
@@ -20,7 +20,7 @@ def _parse_layout_arg(*, layout=None, food=None, bots=None, seed=None):
 
     # prepare layout argument to be passed to pelita.game.run_game
     if layout is None:
-        layout_name, layout_str = get_random_layout(size='normal', seed=seed)
+        layout_name, layout_str = get_random_layout(size='normal', seed=seed, dead_ends=DEAD_ENDS)
```

Comment:
I think `def get_random_layout` should have the dead_end factor already as a default (just as `size="normal"` is a default).

Reply:
Yes, but I thought until we are still testing this allows us to only change `game.DEAD_ENDS`. Hiding it as an argument to `get_random_layout` seems odd to me for a change that is so significant. But we could change it after we have battle tested it and we know that it is going to stay.